### ISSUE TITLE
Crear página para ofertas laborales

### ISSUE THREAD

**Issue by satanas** [pyve/pyve.github.com#76]
Sería ideal algo como una cartelera con las ofertas en vivo pero mientras tanto una lista de recomendaciones es más que suficiente

**Comment by atmantree** [comment]
@sebasmagri hizo una cartelera en vivo [acá](https://github.com/pyve/jobs-on-fire). propones retomar esa idea o hacerlo de otra forma?

**Comment by satanas** [comment]
Si podemos incluir esa cartelera sería genial. Mientras menos reinventemos la rueda, mejor ;)

**Comment by jgomo3** [comment]
Estoy de acuerdo.

Creo que debemos tomar en serio esa aplicación y definir cómo es el proceso para que una oferta aparezca ahí.

¿Tomamos las ofertas de sitios como linked in?¿Establecemos alguna burocracia para que un ofertante logre publicar eso ahí?. Claro, estos serían puntos a tratar en el repo de jobs-on-fire directamente.

**Comment by jgomo3** [comment]
En el footer hay una sección denominada "reclutamiento" que incluye jobs-on-fire.

**Comment by nhomar** [comment]
Me gusta más "Publicar una Oferta de Empleo"

**Comment by jgomo3** [comment]
En python.org tienen una sección llamada "empleos".

**Comment by jgomo3** [comment]
@nhomar Estoy de acuerdo y también creo que deberíamos resaltar más la plataforma nuestra de solicitudes de empleo que la de LinkedIn.

A su vez, tal vez deberíamos deshacernos de nuestra plataforma y utilizar exclusivamente LinkedIn.

A su vez, tal vez debamos hacer que la aplicaciones de empleos también se alimente de las ofertas en LinkedIn :D

**Comment by mamcode** [comment]
> Me gusta más "Publicar una Oferta de Empleo"

Totalmente de acuerdo. Me refiero a cambiar la palabra "Reclutamiento" por "Publicar una Oferta de Empleo".

**Comment by satanas** [comment]
Tenía pensado integrar el código del repo jobs-on-fire aquí (sobre todo su usamos Lektor o Pelican). Sería básicamente un par de modelos y un par de views en backbone. Con eso debería ser suficiente
